app/src/main/res/values/strings.xml

```diff
@@ -188,6 +189,7 @@ like to connect to:</string>
   <string name="sensor_description_internal_storage">Information about the total and available storage space internally</string>
   <string name="sensor_description_last_reboot">The date and time of the devices last reboot</string>
   <string name="sensor_description_light_sensor">The current level of illuminance</string>
+  <string name="sensor_description_location_accurate">Allow Home Assistant to send a notification to request an accurate location along with the application periodically requesting an accurate location.</string>
```

Comment:
Should we also describe the settings the user has control over?

Reply:
Done